common/analysis/lint_rule_status.h

```diff
@@ -127,6 +144,10 @@ struct LintViolation {
 
   const std::vector<AutoFix> autofixes;
 
+  // Additional tokens that are related somehow to
+  // vulnerable token
```

Comment:
would "offending" token be more descriptive?

Reply:
sounds good. changed.